server/app.js

```diff
@@ -137,7 +139,11 @@ module.exports = function createApp({ signInService }, calendarOvertimeService,
 
   // Add services to server
 
-  app.set('DataServices', { calendarService })
+  app.set('DataServices', {
+    calendarService,
+    calendarOvertimeService,
+    notificationService: DEPRECATEnotificationService,
```

Comment:
@nickmcmahon01 see this line here. This is where we can add a toggle between services

Reply:
This does assume that we will have the same method names on each services, I haven't decided if I am going to sync up the method names (cos js be messy like that) or add a syncing function in. Will sort that out on the toggle PR